api/models.py

```diff
@@ -62,3 +62,17 @@ class Task (models.Model):
 	deadline = models.DateTimeField(blank=True, null=True)
 	date_created = models.DateTimeField(auto_now_add=True)
 	date_updated = models.DateTimeField(auto_now=True)
+
+class Event(models.Model):
+
+	title = models.CharField(max_length=255, blank=False, null=False)
+	start_date = models.DateField(blank=False, null=False)
```

Comment:
> it was wrong

@NinaNinchy no, that's not what you did... as you can see [here](https://github.com/NinaNinchy/TrackCat/commit/f773d6c9fe46044fb5929b7ce610b3a94614ddbd) you had all the 4 fields with all of them being DateTimeFields...